packages/react-core/src/demos/DescriptionList/examples/DescriptionListDrawer.tsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import {
+  Gallery,
+  PageSection,
+  GalleryItem,
+  Card,
+  CardBody,
+  Drawer,
+  DrawerPanelContent,
+  DrawerContent,
+  DrawerContentBody,
+  DrawerHead,
+  DrawerActions,
+  DrawerCloseButton,
+  PageSectionVariants,
+  TextContent,
+  Text,
+  DrawerPanelBody,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Button,
+  Tabs,
+  Tab
+} from '@patternfly/react-core';
+import { DashboardWrapper } from '@patternfly/react-core/dist/js/demos/DashboardWrapper';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+
+export const DescriptionListDrawer: React.FunctionComponent = () => {
+  const drawerRef = React.useRef<HTMLDivElement>();
+  const [isExpanded, setIsExpanded] = React.useState(true);
+
+  const onExpand = () => {
+    drawerRef.current && drawerRef.current.focus();
+  };
+
+  const onCloseClick = () => {
+    setIsExpanded(false);
+  };
+
+  const onClickCard = () => {
+    setIsExpanded(true);
+  };
+
+  const panelContent = (
+    <DrawerPanelContent>
+      <DrawerHead>
+        <Text component="h1">ajay-test</Text>
```

Comment:
If we want to match the Core demo,we should use the Title component as an `h2` heading here. Typically there should only be 1 `h1` element accessible on a page (one exception would be in the case of a Modal, since the content outside of a Modal would be hidden from an accessibility standpoint).